exercises/atbash-cipher/src/test/java/AtbashTest.java

```diff
@@ -1,73 +1,59 @@
-@RunWith(Enclosed.class)
 public class AtbashTest {
 
-    @RunWith(Parameterized.class)
-    public static class EncodeTest {
-        private String plaintext;
-        private String ciphertext;
-
-        @Parameters(name = "{index}: expected plaintext \"{0}\" to encode to ciphertext \"{1}\".")
-        public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][]{
-                    {"yes", "bvh"},
-                    {"no", "ml"},
-                    {"OMG", "lnt"},
-                    {"O M G", "lnt"},
-                    {"mindblowingly", "nrmwy oldrm tob"},
-                    {"Testing,1 2 3, testing.", "gvhgr mt123 gvhgr mt"},
-                    {"Truth is fiction.", "gifgs rhurx grlm"},
-                    {"The quick brown fox jumps over the lazy dog.", "gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt"}
-            });
-        }
+    private Atbash atbash;
 
-        public EncodeTest(String plaintext, String ciphertext) {
-            this.plaintext = plaintext;
-            this.ciphertext = ciphertext;
-        }
+    @Before
+    public void setup() {
+        atbash = new Atbash();
+    }
 
+    @Test
+    public void testEncodingSingleWord() {
+        assertEquals("bvh", atbash.encode("yes"));
+        assertEquals("ml", atbash.encode("no"));
+    }
 
-        @Test
-        public void test() {
-            assertEquals(ciphertext, new Atbash().encode(plaintext));
-        }
+    @Ignore("Remove to run test")
+    @Test
+    public void testEncodingWordsWithCapitalCases() {
+        assertEquals("lnt", atbash.encode("OMG"));
     }
 
-    @RunWith(Parameterized.class)
-    public static class DecodeTest {
-        private String ciphertext;
-        private String plaintext;
+    @Ignore("Remove to run test")
+    @Test
+    public void testEncodingWordsWithSpace() {
+        assertEquals("lnt", atbash.encode("O M G"));
+    }
 
-        @Parameters(name = "{index}: expected ciphertext \"{0}\" to decode to plaintext \"{1}\".")
-        public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][]{
-                    {"vcvix rhn", "exercism"},
-                    {"zmlyh gzxov rhlug vmzhg vkkrm thglm v", "anobstacleisoftenasteppingstone"},
-                    {"gvhgr mt123 gvhgr mt", "testing123testing"},
-                    {"gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt", "thequickbrownfoxjumpsoverthelazydog"}
-            });
-        }
+    @Ignore("Remove to run test")
+    @Test
+    public void testEncodingWordsLeadingToCodeWithSpace() {
+        assertEquals("nrmwy oldrm tob", atbash.encode("mindblowingly"));
+    }
 
-        public DecodeTest(String ciphertext, String plaintext) {
-            this.ciphertext = ciphertext;
-            this.plaintext = plaintext;
-        }
+    @Ignore("Remove to run test")
+    @Test
+    public void testEncodingSentences() {
+        assertEquals("gvhgr mt123 gvhgr mt", atbash.encode("Testing,1 2 3, testing."));
```

Comment:
In this case, if someone solving this exercise has a solution which works perfectly apart from that it doesn't encode numbers properly then this test will fail. I think that might not be a very good use experience since the test name is `testEncodingSentences` which doesn't very well point out what the problem is.  So this is one of the reasons I think it would be good to split the tests out to only have one assertion per test 🙂 What do you think? 🙂